views/HakaiWatershedsStreamStations.sql

```diff
@@ -0,0 +1,342 @@
+            turbidity_ntu_ql,
+            turbidity_ntu_qc,
+            turbidity_ntu_uql,
+            turbidity_ntu_med,
+            turbidity_ntu_avg,
+        FROM
+            sn.wsn703_5minute wsn703
+        union
+        SELECT
+            'SSN708US' as station,
+            51.648561 as latitude,
+            -128.06835 as longitude,
+            12 as elevation,
+            measurement_time,
+            record,
+            pls_lvl_ql,
+            pls_lvl_qc,
+            pls_lvl_uql,
+            pls_lvl,
+            pls_lvl_avg,
+            pls_lvl_min,
+            pls_lvl_max,
+            pls_lvl_std,
+            pls_temp_ql,
+            pls_temp_qc,
+            pls_temp_uql,
+            pls_temp,
+            pls_temp_avg,
+            pls_temp_min,
+            pls_temp_max,
+            pls_temp_std,
+            discharge_rate_ql,
+            discharge_rate_qc,
+            discharge_rate_uql,
+            discharge_rate,
+            discharge_rate_min,
+            discharge_rate_max,
+            discharge_volume_ql,
+            discharge_volume_qc,
+            discharge_volume_uql,
+            discharge_volume,
+            discharge_volume_min,
+            discharge_volume_max,
+            turbidity_ql,
+            turbidity_qc,
+            turbidity_uql,
+            turbidity_med,
+            turbidity_avg,
+            turbidity_ntu_ql,
+            turbidity_ntu_qc,
+            turbidity_ntu_uql,
+            turbidity_ntu_med,
+            turbidity_ntu_avg,
+        FROM
+            sn.ssn708us_5minute ssn708us
+        union
+        SELECT
+            'SSN844US' as station,
+            51.66083333 as latitude,
+            -128.0025 as longitude,
+            35 as elevation,
+            measurement_time,
+            record,
+            pls_lvl_ql,
+            pls_lvl_qc,
+            pls_lvl_uql,
+            pls_lvl,
+            pls_lvl_avg,
+            pls_lvl_min,
+            pls_lvl_max,
+            pls_lvl_std,
+            pls_temp_ql,
+            pls_temp_qc,
+            pls_temp_uql,
+            pls_temp,
+            pls_temp_avg,
+            pls_temp_min,
+            pls_temp_max,
+            pls_temp_std,
+            discharge_rate_ql,
+            discharge_rate_qc,
+            discharge_rate_uql,
+            discharge_rate,
+            discharge_rate_min,
+            discharge_rate_max,
+            discharge_volume_ql,
+            discharge_volume_qc,
+            discharge_volume_uql,
+            discharge_volume,
+            discharge_volume_min,
+            discharge_volume_max,
+            turbidity_ql,
+            turbidity_qc,
+            turbidity_uql,
+            turbidity_med,
+            turbidity_avg,
+            turbidity_ntu_ql,
+            turbidity_ntu_qc,
+            turbidity_ntu_uql,
+            turbidity_ntu_med,
+            turbidity_ntu_avg,
+        FROM
+            sn.ssn844us_5minute ssn844us
+        union
```

Comment:
this should be faster with `union all` instead of `union`

Reply:
done